api/controllers/itemController.js

```diff
@@ -1,21 +1,38 @@
-const { itemService } = require("../services");
-const { catchAsync } = require("../utils/error");
+const {itemService} = require("../services");
+const {catchAsync} = require("../utils/error");
+
+const getItems = catchAsync(async (req, res) => {
+    const items = await itemService.getItems(req.query);
+    res.status(200).json({items});
+})
+
+const registerItem = catchAsync(async (req, res) => {
+    const userId = req.user
+    const imageUrl = req.file.location
+    const result = await itemService.registerItem(userId, imageUrl, req.body);
+    if (result === false) {
```

Comment:
```suggestion
    if (!result) {
```
bool 값 판단시에는 일치 연산자를 쓰지 않도록 해주세요.